src/forbes.py

```diff
@@ -0,0 +1,53 @@
+"""
+    This module will take in a json file of billionaires and output the
+    youngest billionaire whose age is above 0. Thereby making you feel crappier
+    about your current income.
+"""
+
+
+import json
+
+
+def age_check():
```

Comment:
This seems like a pretty big function that tries to do a lot of things. Perhaps it could've been broken up into smaller functions that each do one thing.